src/code/z_collision_check.c

```diff
@@ -1,392 +1,3373 @@
 #include <ultra64.h>
 #include <global.h>
+#include <vt.h>
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_collision_check/func_8005B280.s")
+void func_8005B2AC(GraphicsContext* gfx, Vec3f* vA, Vec3f* vB, Vec3f* vC, u8 r, u8 g, u8 b);
 
+// draw red poly
+void func_8005B280(GraphicsContext* gfx, Vec3f* vA, Vec3f* vB, Vec3f* vC) {
+    func_8005B2AC(gfx, vA, vB, vC, 255, 0, 0);
+}
+
+// draw poly
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_collision_check/func_8005B2AC.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_collision_check/func_8005B65C.s")
+s32 Collider_InitBase(GlobalContext* globalCtx, Collider* collider) {
+    static Collider init = { NULL, NULL, NULL, NULL, 0, 0, 0, 0, 3, COLTYPE_INVALID };
+    *collider = init;
+    return 1;
+}
 
-s32 func_8005B6A0(GlobalContext* globalCtx, Collider* collision) {
+s32 Collider_DestroyBase(GlobalContext* globalCtx, Collider* collider) {
     return 1;
 }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_collision_check/func_8005B6B0.s")
+s32 Collider_SetBase_Actor(GlobalContext* globalCtx, Collider* collider, ColliderInit_Actor* src) {
```

Comment:
This is another extra underscore id like to get rid of if possible, along with all the other _Actor ones. I dont know the system well enough to make a suggestion at the moment